x/multisig/model.go

```diff
@@ -0,0 +1,51 @@
+package multisig
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/orm"
+)
+
+// enforce that Contract fulfils desired interface compile-time
+var _ orm.CloneableData = (*Contract)(nil)
+
+// Validate enforces limits of text and title size
+func (c *Contract) Validate() error {
+	return nil
+}
+
+// Copy makes a new Profile with the same data
+func (c *Contract) Copy() orm.CloneableData {
+	return &Contract{
+		Address:             c.Address,
+		Sigs:                c.Sigs,
+		ActivationThreshold: c.ActivationThreshold,
+		ChangeThreshold:     c.ChangeThreshold,
+	}
+}
+
+const ContractBucketName = "contracts"
+
+// ContractBucket is a type-safe wrapper around orm.Bucket
+type ContractBucket struct {
+	orm.Bucket
+}
+
+// NewContractBucket initializes a ContractBucket with default name
+//
+// inherit Get and Save from orm.Bucket
+// add run-time check on Save
+func NewContractBucket() ContractBucket {
```

Comment:
Good use of Bucket. Can you add some helpers to auto-assign id?

Also, we probably will want to allow a query (show all contracts that I am an address of), but as mentioned in tutorial, we still don't support this, so let's add that as a later PR (enhance multisig). And return the new contract address as Data upon creation (please don't lose it;)